### PR TITLE
LPS-142863 Add assignment section

### PR DESCRIPTION
**Pull request by jotascript**
Story: [LPS-139395](https://issues.liferay.com/browse/LPS-139395)

### CODE REVIEW

File: modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/diagram-builder/components/sidebar/sections/EdgeInformation.js [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+						<ClayIcon
+							className="text-muted"
+							symbol="question-circle-full"
+						/>
+					</span>
+				</label>
+
+				<ClayInput
+					id="edgeLabel"
+					onChange={({target}) => {
```
Comment (ramonmsp) [inline modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/diagram-builder/components/sidebar/sections/EdgeInformation.js:79]: @jotascript Is this too much code to onChange? What do you think about change it for two functions, one to set errors and another to set the selected item?

File: modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/diagram-builder/components/sidebar/sections/EdgeInformation.js [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+						<ClayIcon
+							className="text-muted"
+							symbol="question-circle-full"
+						/>
+					</span>
+				</label>
+
+				<ClayInput
+					id="transitionId"
+					onChange={({target}) => {
```
Comment (ramonmsp) [inline modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/diagram-builder/components/sidebar/sections/EdgeInformation.js:150]: @jotascript Almost the same here!

File: modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/diagram-builder/components/sidebar/sections/NodeInformation.js [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+						<ClayIcon
+							className="text-muted"
+							symbol="question-circle-full"
+						/>
+					</span>
+				</label>
+
+				<ClayInput
+					id="nodeLabel"
+					onChange={({target}) => {
```
Comment (ramonmsp) [inline modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/diagram-builder/components/sidebar/sections/NodeInformation.js:52]: @jotascript Same here

File: modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/diagram-builder/components/sidebar/sections/NodeInformation.js [hunk 4]
```diff
@@ -0,0 +1,196 @@
...
+						<ClayIcon
+							className="text-muted"
+							symbol="question-circle-full"
+						/>
+					</span>
+				</label>
+
+				<ClayInput
+					id="nodeId"
+					onChange={({target}) => {
```
Comment (ramonmsp) [inline modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/diagram-builder/components/sidebar/sections/NodeInformation.js:123]: @jotascript Same here

**Review by ramonmsp — CHANGES_REQUESTED** [review]

**Comment by cgmonte** [comment]
Just started reviewing :)

**Comment by cgmonte** [comment]
resent here: https://github.com/liferay-workflow/liferay-portal/pull/715

**Pull request closed without merging**